scenarios/use_client_cert.bdl

```diff
@@ -13,17 +13,17 @@ pool(size = 100,
      worker_type = mqtt_worker,
      worker_start = poisson(100 rps)):
 
-            connect([t(host, "127.0.0.1"),
-                    t(port, 8883),
-                    t(client,fixed_client_id("pool1", worker_id())),
-                    t(clean_session,true),
-                    t(keepalive_interval,60),
-                    t(proto_version,4), t(reconnect_timeout,15), 
-                    t(transport, t(ssl, [t(reuse_sessions, false),
+            connect(host = "127.0.0.1",
+                    port = 8883,
+                    client = fixed_client_id("pool1", worker_id()),
+                    clean_session = true,
+                    keepalive_interval = 60,
+                    proto_version = 4, reconnect_timeout = 15,
+                    transport = t(ssl, [t(reuse_sessions, false),
                                         t(key, load_client_key(resource(keyfile))),
```

Comment:
Is there a reason the 't' tuple form is still in here? can both syntaxes mix?

Reply:
good question,
this form is a shorter way to define proplist, but it works only for function arguments,
it looks a bit inconsistent in this case, what I could suggest is:
1. I can define it to work "standalone", in this case it would be:
```
 transport = t(ssl, (reuse_sessions = false,
                               key = load_client_key(resource(keyfile)),
```
To do that, MZBench has to be updated and the scenario won't work
for older versions, which is not acceptable, but I want to know your opinion

2. Define "id" function, in this case that will look like:
```
 transport = t(ssl, id(reuse_sessions = false,
                                   key = load_client_key(resource(keyfile)),
```
this function can be defined in `worker`,
but it is also better to update MZBench standard library

what do you think?

Reply:
Obviously, the "standalone" version looks cleaner, I'm OK with breaking backward compatibility here, I guess. It's easier to explain to a new user this way. Let me ask though: would the old scenarios still work, in the old proplist syntax? 

But the proposed simplification for proplists is already great...so what I'll do tomorrow is check that the example scripts as they are her run fine with all newest versions.

Reply:
yes, old ones with [t()] will remain valid in any case

Reply:
I have created follow-up issue:
https://github.com/satori-com/mzbench/issues/144

Reply:
thanks, this is going to be great!

Reply:
Coming back to this... this is good to merge? that is the merged syntax works with current MZBench?

Reply:
yes, the one in this PR -- works, the one suggested in discussion -- not yet, I'll PM you about unexpected issue that I got

Reply:
Allright, thanks, then I'll merge this PR for now.

Reply:
Proposed related change to MZBench:
https://github.com/satori-com/mzbench/pull/145